invenio_rdm_records/services/permissions.py

```diff
@@ -82,9 +83,15 @@ class RDMRecordPermissionPolicy(RecordPermissionPolicy):
         IfRestricted("record", then_=can_view, else_=can_all),
     ]
     # Allow reading the files of a record
-    can_read_files = [
+    can_read_files_owner = [RecordOwners(), SecretLinks("view"), SystemProcess()]
```

Comment:
question: why do we add `SecretLinks("view")`? 

I lack a lot of context on permissions so I am just curious 😄

Reply:
Ah this is about secret links 😅 that's what I wasn't understanding.

My question was poorly formulated, sorry!  I understand now what this is about :)